posydon/binary_evol/DT/step_merged.py

```diff
@@ -17,7 +17,7 @@
 from posydon.utils.posydonerror import FlowError
 
 import warnings
```

Comment:
In all files were we use `Pwarn`, we don't need to import `warnings` anymore unless we still have a dedicated use of it. Hence, we can remove it unless there is `warnings.*` anywhere in the file. This saves use from loading same stuff several times.

Reply:
I must have forgotten to remove the import. I very much understand the functionality of how imports and Pwarn work.